tools/storage-explorer/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22
 
 require (
 	github.com/gorilla/mux v1.8.1
-	github.com/onflow/atree v0.8.0-rc.6
+	github.com/onflow/atree v0.8.0
 	github.com/onflow/cadence v1.0.0-preview.52
```

Comment:
Maybe also update to the `v1.0.1` tag?

Reply:
This was just a `go mod tidy` run, the dependencies were already outdated before. We can update to the latest Cadence version in a follow-up PR